### PR TITLE
update compose compiler version to 1.5.13

### PR DESCRIPTION
**Pull request by shishkin-pavel**

### CODE REVIEW

**Review by igordmn — APPROVED** [review]

File: gradle-plugins/gradle.properties [hunk 1]
```diff
@@ -10,7 +10,7 @@ dev.junit.parallel=false
 # Default version of Compose Libraries used by Gradle plugin
 compose.version=1.6.10-dev1590
 # The latest version of Compose Compiler used by Gradle plugin. Used only in tests/CI.
-compose.tests.compiler.version=1.5.10.2
+compose.tests.compiler.version=1.5.12
 # The latest version of Kotlin compatible with compose.tests.compiler.version. Used only in tests/CI.
 compose.tests.compiler.compatible.kotlin.version=1.9.23
```
Comment (igordmn) [inline gradle-plugins/gradle.properties:15]: It seems it should be 1.9.24

File: compose/integrations/composable-test-cases/gradle.properties [hunk 2]
```diff
@@ -5,15 +5,15 @@ kotlin.version=1.9.23
 agp.version=7.3.0
 
 # a version of compose libraries. In this project the only dependency is compose-runtime.
-compose.version=1.6.1
+compose.version=1.6.10-beta02
```
Comment (MatkovIvan) [inline compose/integrations/composable-test-cases/gradle.properties:8]:
```suggestion
compose.version=1.6.10-rc01
```

Comment (shishkin-pavel) [reply compose/integrations/composable-test-cases/gradle.properties]: fixed

File: gradle-plugins/gradle.properties (hunk 1 above)
Comment (shishkin-pavel) [reply gradle-plugins/gradle.properties]: fixed

**Review by igordmn — APPROVED** [review]

File: gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt [hunk 3]
```diff
@@ -23,7 +23,8 @@ internal object ComposeCompilerCompatibility {
         "1.9.20" to "1.5.3",
         "1.9.21" to "1.5.4",
         "1.9.22" to "1.5.8.1",
-        "1.9.23" to "1.5.10.2",
+        "1.9.23" to "1.5.12",
```
[the remark this answers is not in the document]
Comment (shishkin-pavel) [reply gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt:26]: jetpack compose 1.5.12 is compatible with 1.9.23, so it is probably a good thing to have our 1.5.12 compatible with it also
https://developer.android.com/jetpack/androidx/releases/compose-kotlin#pre-release_kotlin_compatibility
also, Ive noted that 1.5.13 is up, so probably we should abandon these pulls and support 1.5.13 for both 1.9.23 and 1.9.24?

File: gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt [hunk 4]
```diff
@@ -23,7 +23,8 @@ internal object ComposeCompilerCompatibility {
         "1.9.20" to "1.5.3",
         "1.9.21" to "1.5.4",
         "1.9.22" to "1.5.8.1",
-        "1.9.23" to "1.5.10.2",
+        "1.9.23" to "1.5.13",
```
Comment (eymar) [inline gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt:26]: It looks suspicious. Usually one version of compose compiler supports only one version of kotlin.
Or did you disable the version checker in the Compose Compiler plugin code?

Comment (shishkin-pavel) [reply gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt:26]: you are definitely right, fixed

**Review by eymar — APPROVED** [review]

**Pull request merged**